frontend/src/main/components/Sections/SectionsTable.js

```diff
@@ -246,14 +246,14 @@ export default function SectionsTable({ sections }) {
       aggregate: getFirstVal,
       Aggregated: renderInfoLink,
     },
-    // Stryker disable all : difficult to test modal interaction but we should try in future work
+    // Sssssstryker disable all : difficult to test modal interaction but we should try in future work
```

Comment:
```suggestion
```

Nice work on removing the stryker exception!  Can we just go ahead and remove these comments, though?